src/goto-programs/goto_convert_functions.cpp

```diff
@@ -151,7 +151,13 @@ void goto_convert_functionst::convert_function(
   // make tmp variables local to function
   tmp_symbol_prefix=id2string(symbol.name)+"::$tmp";
 
-  f.type=to_code_type(symbol.type);
+  // store the parameter identifiers in the goto functions
+  const code_typet &code_type = to_code_type(symbol.type);
+  f.type = code_type;
+  f.parameter_identifiers.clear();
+  f.parameter_identifiers.reserve(code_type.parameters().size());
+  for(const auto &parameter : code_type.parameters())
```

Comment:
We have this loop to copy parameters from a `code_typet` to `parameter_identifiers` in at least 4 different places. Maybe add a helper function that does that?

Reply:
Indeed - I have now added a method `goto_functiont::set_parameter_identifiers(const code_typet&)`.